pages/register.css

```diff
@@ -0,0 +1,109 @@
+.container {
+    height: 500px;
+    width: 900px;
+    background-color: var(--color-3);
+    border-radius: 15px;
+    border: 1px solid var(--color-1);
+    padding: 30px 20px 30px;
+}
+.some-text {
+    height: 50px;
+    width: 200px;
```

Comment:
ai nevoie de width? Din cate am vazut 'Register' e singur pe linie.
Ai putea de asemenea sa schimbi 'some-text' cu 'register-title' sau 'register'

Reply:
eu nici nu aveam some-text in html.. aparent cred ca am schimbat la un moment dat si nu mi-am dat seama am sters toate acele linii de la some-text din css pt ca nu mai aveam nevoie de ele